release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -245,7 +245,7 @@ For more information, see xref:../edge_computing/cnf-talm-for-cluster-upgrades.a
 [id="ocp-4-16-sha-haproxy-support-removed"]
 === SHA1 certificates no longer supported for use with HAProxy
```

Comment:
Supposedly, SHA-1 is the proper spelling for SHA1. I didn't know this until recently. For example, https://en.wikipedia.org/wiki/SHA-1. 
 
```suggestion
=== SHA-1 certificates no longer supported for use with HAProxy
```

Reply:
ty